src/Makefile

```diff
@@ -19,18 +19,28 @@ OUT_SHARED_NO = ../lib/libnophiprof.so
 CLOCK_ID = CLOCK_MONOTONIC
 
 # C++ compiler flags 
-CCFLAGS = -fpic -O2 -fopenmp -std=c++17 -DCLOCK_ID=$(CLOCK_ID) -W -Wall -Wextra -pedantic 
+CCFLAGS = -fpic -O2 -std=c++17 -DCLOCK_ID=$(CLOCK_ID) #-W -Wall -Wextra -pedantic 
```

Comment:
Should re-enable warnings. Now that it compiles cleanly we want to make sure it continues to. 

GCC with openmp and no nvtx should be standard. We need a better way (configure ?) for handling different platforms.

Reply:
Ok, I re-structured the makefile to allow different flags for different compiler environments. NVTX is default on for PGI. Also updated the Readme.